### ISSUE TITLE
Microlensing in twinkles

### ISSUE THREAD

**Issue by tcollett** [DarkEnergyScienceCollaboration/Twinkles#286]
Twinkles 2 probably needs to include microlensing for the lensed quasars and supernovae

**Comment by drphilmarshall** [comment]
Agreed. Timo is helping with the microlensing for TDC2, I'm thinking we
should try and make the Twinkles 2 lenses use the same once. @danielsf, we
should talk about this in Tucson and/or at the retreat.

**Comment by danielsf** [comment]
@drphilmarshall Sounds good.  See you in Tucson.
